.gitignore

```diff
@@ -2,4 +2,6 @@
 *.jl.*.cov
 *.jl.mem
 stratch/*
-_init.jl
\ No newline at end of file
+_init.jl
+*.sublime-workspace
+*.sublime-project
```

Comment:
These should really be in your global ignore file: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore